add serialization (#376)
nervosnetwork/docs.nervos.org: website/docs/serialization/features.mdx

---
id: features-molecule

> title: Features

Reviewer: ```suggestion
title: Rust Library Features
```

---

import Tooltip from "@components/Tooltip";

> # Features

Reviewer: ```suggestion
# Rust Library Features
```

> The `std` feature, contained in `default`, is enabled by default and comprises:

Reviewer: Molecule is developed in Rust language. The `default` feature in the Rust library is enabled by default, including `std` feature and more:

> The `std` feature, contained in `default`, is enabled by default and comprises:

Reviewer: ```suggestion
Molecule is developed in Rust language. The `default` feature in the Rust library is enabled by default, including `std` feature and more:
```
